From ea0f8a4df792f36dd7339e5342291ec5319ec3c3 Mon Sep 17 00:00:00 2001
From: Stefan Roese <sr@denx.de>
Date: Thu, 12 Sep 2019 07:51:36 +0200
Subject: [PATCH 2/3] ubi: Print skip_check in ubi_dump_vol_info()

It might be interesting, if "skip_check" is set of not, so lets print
this flag in ubi_dump_vol_info() as well.

Signed-off-by: Stefan Roese <sr@denx.de>
Cc: Quentin Schulz <quentin.schulz@bootlin.com>
Cc: Boris Brezillon <boris.brezillon@bootlin.com>
---
 drivers/mtd/ubi/debug.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/ubi/debug.c b/drivers/mtd/ubi/debug.c
index 0a7427522c..f3d348da83 100644
--- a/drivers/mtd/ubi/debug.c
+++ b/drivers/mtd/ubi/debug.c
@@ -109,6 +109,7 @@ void ubi_dump_vol_info(const struct ubi_volume *vol)
 	printf("\tlast_eb_bytes   %d\n", vol->last_eb_bytes);
 	printf("\tcorrupted       %d\n", vol->corrupted);
 	printf("\tupd_marker      %d\n", vol->upd_marker);
+	printf("\tskip_check      %d\n", vol->skip_check);
 
 	if (vol->name_len <= UBI_VOL_NAME_MAX &&
 	    strnlen(vol->name, vol->name_len + 1) == vol->name_len) {
-- 
2.23.0

